tests/migrations/test_operations.py

```diff
@@ -2068,6 +2076,52 @@ def test_remove_field(self):
         self.assertEqual(definition[1], [])
         self.assertEqual(definition[2], {"model_name": "Pony", "name": "pink"})
 
+    def test_remove_field_blocked(self):
+        """
+        Tests migration is blocked by dependent objects.
+        """
+        project_state = self.set_up_test_model("test_rmflb")
+        new_state = project_state.clone()
+
+        with connection.cursor() as cursor:
+            cursor.execute(
+                "CREATE VIEW test_rmflb_pony_view AS SELECT pink FROM test_rmflb_pony"
+            )
+            cursor.execute("SELECT * FROM test_rmflb_pony_view")
```

Comment:
We could add some `assertViewExists` and `assertViewNotExists` methods something like:
```python
    def assertViewExists(self, view_name):
        with connection.cursor() as cursor:
            cursor.execute(f"SELECT * FROM {view_name}")

    def assertViewNotExists(self, view_name):
        with self.assertRaises(OperationalError):
            self.assertViewExists(view_name)
```
and then later the asserts look like:
```python
       # Positive
        self.assertViewExists(view_name)
        self.assertColumnExists("test_rmflb_pony", "pink")
        # Negative
        self.assertViewNotExists(view_name)
        self.assertColumnNotExists("test_rmflb_pony", "pink")
```